packages/react-accordion/src/components/AccordionHeader/useAccordionHeader.ts

```diff
@@ -27,10 +27,12 @@ export const useAccordionHeader = (props: AccordionHeaderProps, ref: React.Ref<H
   const innerRef = React.useRef<HTMLElement>(null);
 
   const buttonShorthand = useARIAButton(props.button, {
-    id,
-    disabled,
-    // 'aria-controls': panel?.id,
-    children: React.Fragment,
+    optional: false,
+    defaultProps: {
+      id,
+      disabled,
+      // 'aria-controls': panel?.id,
```

Comment:
nit: can you say why this is commented out ?

Reply:
I'll remove it honestly. this is not working currently. A mechanism for sharing `id` between composed components should be implemented